### PR TITLE
Provisioning component and linting CSS

### PR DESCRIPTION
**Pull request by Ericat**
First provisioning, may have messed up the commit tags a bit, but should be good to go. Please check version `0.0.0-development`, that looks strange to me

### CODE REVIEW

**Comment by keithamus** [comment]
It looks like you've got a copy of the whole source added into a subfolder. My guess is you ran `provision-react-component component-beta-bar` and it has made a new component in that folder. You'll need to remove all of that.

**Comment by keithamus** [comment]
In fact if you just rebase and remove `b946e80` commit I think that'll be enough :wink:

File: src/betabar.css [hunk 1]
```diff
@@ -27,22 +28,29 @@
...
+  background: var(--betabar-foreground-color, var(--color-thimphu));
 }
 
 .beta-bar--feedback:hover {
   background: var(--betabar-feedback-hover, var(--color-boston));
 }
 
+/* stylelint-disable selector-no-type */
+
 .beta-bar--close-wrapper:hover svg {
```
Comment (umbobabo) [inline src/betabar.css]: I think you should have a className applied to the svg that you can use to not disable the linter rule. NTH.

**Comment by Ericat** [comment]
@keithamus yes I did that first then reverted back, I feel like I forgot to git clean first. Will do!

File: src/betabar.css (hunk 1 above)
Comment (Ericat) [inline src/betabar.css]: The issue is that the icon is the close button, which is just imported but not in the source code...so I can't see a way to pass down a prop from here. I'd rather just go ahead and merge this if it looks good, so I can carry on with fe-blogs. Is that ok?

**Pull request closed without merging**